ketcher-autotests/tests/File-Management/Open-And-Save-Files/InChi/InChi.spec.ts

```diff
@@ -151,3 +157,353 @@ test.describe('', () => {
     },
   );
 });
+
+test.describe('Open and Save InChI file', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  const WAIT_TIMEOUT = 5000;
```

Comment:
is there a reason to add timeouts all over?

Reply:
yes, without timeouts the inchi file format isn't browsing